neptunelib/session.py

```diff
@@ -25,22 +25,33 @@ class Session(object):
     In order to query Neptune experiment database in any way you have to instantiate this object first.
 
     Attributes:
-        credentials (:obj:`Credentials`): Description of `attr2`.
+        api_token(str): This is a secret API key that you can retrieve by running `neptune account api-token get <org>`.
```

Comment:
It's actually `neptune account api-token get` (no `<org>`). The docstring was inaccurate.